source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -312,6 +312,9 @@
     getDashboardEvaluations(window.ViewData.period);
     ko.applyBindings(viewmodel);
     commonSuggest($('.content-modal .search'), 'UserName');
+    $("#selectedPeriod").change(function () {
+        window.location = "/Evaluations/" + this.value;
```

Comment:
In my opinion it is better to set the evaluation URL as values in the combobox (see [line 95](https://github.com/CommonJobs/CommonJobs/pull/476/files#diff-48f065f3d88335da2dce595120021376R95)), it would be better to avoid to hardcode URL sections in JavaScript.

But maybe is there a technical impediment, did you try my suggested approach?